packages/lesswrong/lib/collections/users/helpers.ts

```diff
@@ -97,6 +97,12 @@ Users.canModeratePost = (user: UsersMinimumInfo|DbUser|null, post: PostsBase|DbP
   )
 }
 
+Users.canModerateComment = (user: UsersMinimumInfo|DbUser|null, post: PostsBase|DbPost|null , comment: CommentsList|DbComment) => {
```

Comment:
I am currently worried this falls open when you don't provide a post. I think you should error or return false when you fail to provide any of the three arguments. Though that itself might run into problems with Jim's recent PR, but we can jump off that bridge when we get to it.